env-fenics.sh

```diff
@@ -3,8 +3,8 @@ source ${HOME}/fenicsx-iris-cluster/env-build-fenics.sh
 
 # Note: These must be unset initially, need to code this in somehow.
 export PETSC_DIR=${PREFIX}
-export DOLFINX_JIT_CACHE_DIR=${PREFIX}/cache
-export DOLFINX_JIT_CFLAGS="-O3 -march=native"
+mkdir -p ${HOME}/.config/dolfinx
+echo '{"cache_dir":"'${PREFIX}/.cache/fenics'", "cffi_extra_compile_args": ["-O3", "-march=haswell"]}' > ${HOME}/.config/dolfinx/dolfinx_jit_parameters.json
```

Comment:
What happens here if this file is sourced on multiple nodes?

Reply:
I expect all are touching it. I didn't see any lock or hang, this must be somehow handled in the way how nodes access the $SCRATCH. Feel free to add a condition and `echo` on first node only.